addons/survey/views/survey_survey_views.xml

```diff
@@ -83,68 +84,68 @@
-                                    <div attrs="{'invisible': [('certification', '=', False)]}" class="d-flex">
-                                        <field name="certification_report_layout" class="oe_inline"/>
-                                        <button name="action_survey_preview_certification_template"
-                                            string="Preview" type="object"
-                                            icon="fa-external-link"  target="_blank" class="btn-link pt-0"/>
+                                    <div colspan="2" class="o_checkbox_optional_field">
+                                        <label for="certification"/>
+                                        <field name="certification" nolabel="1"/>
+                                        <div attrs="{'invisible': [('certification', '=', False)]}" class="w-100">
+                                            <field name="certification_report_layout" class="w-50"/>
+                                            <button name="action_survey_preview_certification_template"
+                                                string="Preview" type="object"
+                                                icon="fa-external-link"  target="_blank" class="btn-link pt-0"/>
+                                        </div>
+                                    </div> 
+                                    <field name="certification_mail_template_id"/>
+                                    <div colspan="2" class="float-left o_checkbox_optional_field" attrs="{'invisible': ['|', ('certification', '=', False), ('users_login_required', '=', False)]}">
+                                        <label for="certification_give_badge"/>
+                                        <field name="certification_give_badge"/>
+                                        <div attrs="{'invisible': [('certification_give_badge', '=', False)]}">
+                                            <field name="certification_badge_id"
+                                                attrs="{'invisible': ['|', ('certification_give_badge', '=', False), ('certification_badge_id', '!=', False)], 'required': [('certification_give_badge', '=', True)]}"
+                                                domain="[('survey_id', '=', active_id), ('survey_id', '!=', False)]"
+                                                context="{'default_name': title,
+                                                        'default_description': 'Congratulations, you have succeeded this certification',
+                                                        'default_rule_auth': 'nobody',
+                                                        'default_level': None,
+                                                        'form_view_ref': 'survey.gamification_badge_form_view_simplified',
+                                                        'default_website_published': True}"/>
```

Comment:
Why would be publish it by default ? Especially this is valid only in website_profile module ... would remove it, unless you really want it ?

Edit: ok, this is a move .. let us keep it then, nobody complained.